Production/govuk_ios/ViewModels/AppErrorViewModel.swift

```diff
@@ -25,7 +25,7 @@ struct AppErrorViewModel {
             body: String.common.localized("genericErrorBody"),
             buttonTitle: String.common.localized("genericErrorButtonTitle"),
             buttonAccessibilityLabel: String.common.localized(
-                "genericErrorTitleAccessibilityLabel"
+                "genericErrorButtonTitleAccessibilityLabel"
```

Comment:
Does this need changing in the code as well?

Reply:
Was just a mismatch between this and the strings file key, so have changed it to match the key.  But I missed it in the tests, so I need another review 🙄